src/popup.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link rel="stylesheet" type="text/css" href="popup.css">
+  </head>
+
+  <body>
+    <div id="dgtw-status"></div>
```

Comment:
One tiny change - I'd like to see some spacing between the status message and the controls. You may be able to accomplish this satisfactorily just by using `<p>` rather than `<div>`. If that's not satisfactory, there's always the CSS.